terraso_backend/apps/graphql/schema/story_maps.py

```diff
@@ -94,12 +131,170 @@ def mutate_and_get_payload(cls, root, info, **kwargs):
         user = info.context.user
         story_map = StoryMap.objects.get(pk=kwargs["id"])
 
-        if not user.has_perm(StoryMap.get_perm("delete"), obj=story_map):
+        if not rules.test_rule("allowed_to_delete_story_map", user, story_map):
```

Comment:
question: I guess the idea here is that we should `Model.get_perm` if the permissions line up with the create, edit delete rules, and otherwise just use the `test_rule`? I ask because I think the current `Project` class should be doing the second one then.

Reply:
I did some research for this and found that `has_perm` behaves diferently, it seems it actualy links to permissions that can be modified in the DB and assigned to users, so this was actually not working, with `rules.test_rule` I was sure that the validation was triggered every time, so still not completely sure about how `user.has_perm` works but with `rules.test_rule` it is calling what I need

Reply:
Cool, good to know!